project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   object V {
     val cats         = "2.6.1"
     val circe        = "0.14.1"
-    val igluClient   = "1.1.1"
+    val igluClient   = "1.4.0-M3"
```

Comment:
milestone release used here - intentional?

Reply:
Yep, I will replace it once we release final version.